src/chunk_insert_state.c

```diff
@@ -535,6 +542,25 @@ ts_chunk_insert_state_create(Chunk *chunk, ChunkDispatch *dispatch)
 	return state;
 }
 
+void
```

Comment:
Do we  have to call this function in copy.c also after ts_chunk_dispatch_get_chunk_insert_state?